src/index.js

```diff
@@ -82,6 +82,10 @@ export default class Voices {
     return this.byLangCode('pl-PL')
   }
 
+  spanish () {
+    return this.byLangCode('es-ES')
+  }
+
```

Comment:
For Spanish language in general you should use the method `byLang`. Take a look at `portuguese` or `english` method.

In this way you are getting only `Castilian Spanish`